.eslintrc.js

```diff
@@ -155,9 +155,13 @@ module.exports = {
         selector: "CallExpression[callee.object.name='assert'][callee.property.name='doesNotThrow']",
         message: "Please replace `assert.doesNotThrow()` and add a comment next to the code instead."
       },
+      {
+        selector: `CallExpression[callee.object.name='assert'][callee.property.name='rejects'][arguments.length<2]`,
+        message: 'assert.rejects() must be invoked with at least two arguments.',
+      },
       {
         selector: `CallExpression[callee.object.name='assert'][callee.property.name='throws'][arguments.1.type='Literal']:not([arguments.1.regex])`,
-        message: 'use a regular expression for second argument of assert.throws()',
+        message: 'Use a object as second argument of assert.throws()',
```

Comment:
an object

Reply:
This is a change from #19885 that I plan on removing in here as soon as that lands. I already fixed that in there though.